nym-connect/desktop/scripts/pre_meduim_build.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+set -e
+
+if ! [[ "$RELEASE_TAG" =~ ^nym-connect-s-v.*? ]]; then
+  echo -e " ✗ Invalid release tag $RELEASE_TAG"
+  exit 1
+fi
+
+version="${RELEASE_TAG#nym-connect-s-v}"
+
+sed -i "s/^name = \".*\"/name = \"nym-connect-s\"/" src-tauri/Cargo.toml
+sed -i "s/^version = \".*\"/version = \"$version\"/" src-tauri/Cargo.toml
+sed -i 's/"productName": "nym-connect"/"productName": "NymConnect S"/' src-tauri/tauri.conf.json
```

Comment:
will this cause issues? (the different capitalization)

Reply:
I don't think, in the [doc](https://tauri.app/v1/api/config/#packageconfig) there are no notes regarding name convention